fedimint-ui/apps/guardian-ui/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "guardian-ui",
```

Comment:
Should we namespace all this repo's packages with `@fedimint/*`? Maybe we should reserve that npm organization now too?

Reply:
This is especially valuable for package name collisions, e.g. `tsconfig` is a match for https://www.npmjs.com/package/tsconfig which could be confusing.

Reply:
ah great idea. agreed on namespacing

Reply:
reserved https://www.npmjs.com/org/fedimint

Reply:
I'll make a follow up PR that nameplates these packages